### PR TITLE
check language and show localization messages

### PR DESCRIPTION
**Pull request by victorhck**
the original script didn't work for me, because git show messages in spanish.
This script, check $LANG variable and show different messages (now only english and spanish)
I haven't tested in English, can you please check if everything is ok in your system?
Greetings!!

### CODE REVIEW

File: git-rundown.sh [hunk 1]
```diff
@@ -37,31 +63,31 @@ do
 		echo -en "\033[0;36m${s:10}\033[0m "
 
 		# Check for modified files
-		if [ $(git status | grep modified -c) -ne 0 ]
+		if [ $(git status | grep $check_modif -c) -ne 0 ]
```
Comment (victoriadrake) [inline git-rundown.sh]: Each variable needs double quotes throughout the script:

```suggestion
		if [ $(git status | grep "$check_modif" -c) -ne 0 ]
```
```

**Review by victoriadrake — CHANGES_REQUESTED** [review]
Hello! Thank you for this translation, I love this idea!

I have some errors now in the English version output:

```sh
/home/v/src/ghub/git-rundown
master Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
./git-rundown/multilang.sh: line 66: [: -ne: unary operator expected
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
./git-rundown/multilang.sh: line 73: [: -ne: unary operator expected
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
./git-rundown/multilang.sh: line 80: [: -ne: unary operator expected
```

I think it is because the new variables need to be double quoted. Please see the review comment.

**Comment by victorhck** [comment]
it's strange that works for Spanish and not in English... don't know why.
A new PR to test if everything now runs ok.

**Comment by victoriadrake** [comment]
I realized the issue...

```sh
$ echo $LANG
en_US.UTF-8
```

Quick fix! If there is enough interest, the script may want to handle other locales as well.

**Comment by victoriadrake** [comment]
Thank you @victorhck!

**Comment by victorhck** [comment]
> 
> 
> Thank you @victorhck!

Thanks to you!! Was a pleasure to learn, hack a bit and contribute in your code! :)
Greetings!
PS: Maybe I'll blog about the script...

**Comment by victoriadrake** [comment]
Please do! :+1: I'd love if you dropped a link here too!

**Comment by victorhck** [comment]
> Please do! +1 I'd love if you dropped a link here too!

here you have: 
https://victorhckinthefreeworld.com/2020/03/05/comprueba-el-estado-de-multiples-repositorios-de-git-con-este-script/

even a asciinema demo:
https://asciinema.org/a/307752

:)

**Comment by victoriadrake** [comment]
Very cool! ¡Muchas gracias por compartir!

**Pull request merged**